packages/core/src/local-workspace/state.ts

```
@@ -96,7 +96,11 @@ export const localState = (filePath: string): state.State => {
     },
     rename: async (name: string): Promise<void> => {
       const newFilePath = path.join(path.dirname(currentFilePath), `${name}${ZIPPED_STATE_EXTENSION}`)
-      await rename(currentFilePath, newFilePath)
+      if (exists(currentFilePath)) {
+        await rename(currentFilePath, newFilePath)
+      } else {
+        log.debug(`Rename failed. ${currentFilePath} Does not exists`)
```

is there a reason we want to suppress the error we used to have in this case?
feels like if we fail to rename the state we should not treat it as a successful rename

---

@noamhammer @adamro In our original discussion we have reached a decision that our rename wrappers should be "forgiving", though at the time I haven't understood fully our arguments. Have some thoughts?